frontend/src/components/List/Column/ImplementationColumn.tsx

```diff
@@ -46,6 +47,7 @@ const ICONS = {
   Totem: totemIcon,
   'Nodle Chain Node': nodleIcon,
   subzero: zeroIcon,
+  'Crust Network CC2': crustIcon,
```

Comment:
```suggestion
  Crust: crustIcon,
```
It should match the name of the node, not the chain:
![Screenshot from 2020-12-10 09-36-36](https://user-images.githubusercontent.com/1096222/101745208-6c057080-3acb-11eb-941f-10655a64f19f.png)

Reply:
Thank you. I have fixed this.